packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -95,10 +95,13 @@ export function makeVatKeeper(
     kvStore.set(`${vatID}.options`, JSON.stringify(options));
   }
 
-  function getSourceAndOptions() {
+  /**
+   * @param {boolean} consensusMode
+   */
+  function getSourceAndOptions(consensusMode) {
```

Comment:
I'm not jazzed about the idea of passing override options *into* a function like this.. I think it'd be better to do the override at the point of use, and leave the vatKeeper methods as purely "what does the DB say?".